src/db/migrations/02__MessageStateTransitionTable.ts

```diff
@@ -0,0 +1,36 @@
+import { QueryInterface, DataTypes } from 'sequelize';
```

Comment:
Hmmm.. Do you think this could have actually been added to the db.ts file directly since it's just a new table?
Happy to keep it here for now to prevent churn, but next time I might test that out. Better than having some db stuff be declared here and some there.